src/System.IO.FileSystem.AccessControl/ref/System.IO.FileSystem.AccessControl.cs

```diff
@@ -10,6 +10,7 @@ namespace System.IO
     public static partial class FileSystemAclExtensions
     {
         public static void Create(this System.IO.DirectoryInfo directoryInfo, System.Security.AccessControl.DirectorySecurity directorySecurity) { }
+        public static System.IO.FileStream Create(this System.IO.FileInfo fileInfo, System.IO.FileMode mode, System.Security.AccessControl.FileSystemRights rights, System.IO.FileShare share, int bufferSize, System.IO.FileOptions options, System.Security.AccessControl.FileSecurity fileSecurity) { throw null; }
```

Comment:
Does the order matter? In the API Proposal, the file Create method was above the directory Create method.

Reply:
Shouldn't.